test/types.documentarray.test.js

```diff
@@ -748,4 +748,18 @@ describe('types.documentarray', function() {
     doc.arr.push(subdoc);
     await doc.validate();
   });
+
+  it('applies _id default (gh-12264)', function() {
+    mongoose.deleteModel(/Test/);
+    const nestedArraySchema = Schema({
+      subDocArray: [{ name: String }]
+    });
+
+    const Model = db.model('Test', nestedArraySchema);
+    const doc = new Model().init({
```

Comment:
Is `new Model().init(...)` the equivalent of creating a document in the DB by bypassing mongoose and then fetching it?

Reply:
`init()` is what Mongoose calls to set a document's properties when loading it from the database. So yeah.